openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, ocp_cluster: Dict[str, Any], user_input: UserInput) -> None:
             self.openshift_install_binary_path = ""
             self.ipi_base_available_versions: Dict[str, Dict[str, List[str]]]
             self.cluster["ocm-env"] = self.cluster_info["ocm-env"] = PRODUCTION_STR
-        if self.cluster_info["fips"]:
+        if self.cluster_info.get("fips"):
```

Comment:
```
if self.fips := self.cluster_info.get("fips") :
    os.environ["OPENSHIFT_INSTALL_SKIP_HOSTCRYPT_VALIDATION"] = "true"
```

and use `self.fips`

Reply:
I can't use `:=` with a `self.` variable. I have modified it slightly and pushed changes.